app/assets/javascripts/subNav.js

```diff
@@ -0,0 +1,29 @@
+(function(window) {
+    "use strict";
+
+    document.addEventListener('DOMContentLoaded', function() {
+        const sideNavLinks = document.querySelectorAll('.usa-sidenav__item');
+        const currentURL = window.location.pathname;
+
+        sideNavLinks.forEach(link => {
+            const sublist = link.querySelector('.usa-sidenav__sublist');
+            const parentLink = link.querySelector('a');
+
+            const isMatch = parentLink && sublist && currentURL === parentLink.getAttribute('href');
+            if (isMatch) {
+                sublist.classList.replace('display-none', 'display-block');
+            }
+
+            link.addEventListener('click', function(event) {
+                if (sublist && event.target === parentLink) {
+                    event.preventDefault();
+
+                    sublist.classList.toggle('display-none');
+                    sublist.classList.toggle('display-block');
+
+                    setTimeout(() => window.location.href = parentLink.href, 50);
+                }
+            });
+        });
+    });
+})(window);
```

Comment:
This is very clean, thank you!  However, I'm wondering a bit if it's necessary.

Am I understanding this correctly that it's handling CSS toggles on menu items?  Is it at all possible this could be handled via CSS directly?  I might be naively thinking about this and that's totally fine if not!  I just wanted to check to be sure. 🙂

Reply:
I changed the submenu to expand on hover rather than on click, which simplifies the code. We can gather user feedback on their preference for the side menu.